test/e2e-wi-tests.sh

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
+#chmod +x e2e-wi-tests.sh
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname $0)/e2e-common.sh
+
+# Override the setup and teardown functions to install wi-enabled control plane components.
+
+readonly CONTROLLER_SERVICE_ACCOUNT="controller"
```

Comment:
To avoid confusion, could you please rename it to `K8S_CONTROLLER_SERVICE_ACCOUNT`?